sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
+
+\subsection{Researchers and staff affiliated with Flemish university associations}
+
+\begin{itemize}
+  \item Includes externally funded researchers registered in the personnel database (FWO, SBO, VIB, IMEC, etc.)
+  \item Includes researchers from all VSC partners
```

Comment:
mention list of VSC partners?

Reply:
Nah, that would mean listing the associations (and they keep changing their names)

Reply:
OK, fine